shell/platform/windows/platform_handler_winuwp.cc

```diff
@@ -21,24 +28,81 @@ PlatformHandlerWinUwp::PlatformHandlerWinUwp(BinaryMessenger* messenger,
 
 PlatformHandlerWinUwp::~PlatformHandlerWinUwp() = default;
 
+bool isWindowInForeground() {
+  auto activation_mode =
+      winrt::Windows::UI::Core::CoreWindow::GetForCurrentThread()
+          .ActivationMode();
+  return activation_mode == winrt::Windows::UI::Core::CoreWindowActivationMode::
+                                ActivatedInForeground;
+}
+
 void PlatformHandlerWinUwp::GetPlainText(
     std::unique_ptr<MethodResult<rapidjson::Document>> result,
     std::string_view key) {
-  // TODO: Implement. See https://github.com/flutter/flutter/issues/70214.
-  result->NotImplemented();
+  // We call `Clipboard::GetContent()` when the application window is in
+  // focus, otherwise calling this will throw an error.
+  if (isWindowInForeground()) {
+    auto content =
+        winrt::Windows::ApplicationModel::DataTransfer::Clipboard::GetContent();
+    // Calling `DataPackageView::GetTextAsync()` when the clipboard has no text
+    // content will throw an error.
+    if (content.Contains(winrt::Windows::ApplicationModel::DataTransfer::
+                             StandardDataFormats::Text())) {
+      // Waiting `DataPackageView::GetTextAsync()` using `TResult.get()` on the
+      // platform thread will causes the application stop, so we continue
+      // response on this callback.
+      content.GetTextAsync().Completed([result = std::move(result), key](
```

Comment:
Not super familiar with these APIs, but, does this have an error handler as well that we may need to account for? Perhaps one that also gets invoked when the window is not in the foreground.

Reply:
Before here I check window is in foreground, so I think this should throw no errors. I created an example and checked app doesn't crash from the engine. (I will post it on the comments.)